storybook/src/components/DescriptionList/DescriptionList.docs.mdx

```diff
@@ -20,6 +20,14 @@ A term may have multiple descriptions.
 
 <Canvas of={DescriptionListStories.MultipleDescriptions} />
 
+### Multiple terms
+
+In the case of multiple consequtive terms for a description you need to
+wrap all `DescriptionList.Term` and `DescriptionList.Description` pairs in `DescriptionList.Group`.
+And it will also require the minWidth property to be set to anything but `auto` to prevent the misalignment of different groups.
```

Comment:
```suggestion
If multiple terms share a single description, group them in a `DescriptionList.Group` to ensure proper grid layout.
Note that this approach is incompatible with the automatic width of the terms column; select a fixed width option instead.
```

Reply:
> Wait a minute, we can adjust the grid column widths automatically if a group is present 🤯🎉
> 
> ```css
> .ams-description-list__group:has(.ams-description-list__group) {
>   grid-template-columns: 1fr 2fr;
> }
> ```
> 
> Maybe as a separate declaration block to prevent invalidating the entire block for browsers that don’t recognise the `:has` pseudo-class?

`.ams-description-list__group:has(.ams-description-list__group)` Wouldn't work, it implies that the group contains another group. The first column width is something I'm not happy about, i'll check if it works without breakpoints.

Reply:
Sorry big typo of course – I meant

```
.ams-description-list:has(.ams-description-list__group) {
  grid-template-columns: 1fr 2fr;
}
```

“If the list has a group, apply the 33/67 column widths to it instead of auto.”